test/nodejs-bundle/circuit/dial-over-any-relay.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env mocha */
+'use strict'
+
+const TCP = require('libp2p-tcp')
+const WS = require('libp2p-websockets')
+const multiplex = require('libp2p-multiplex')
+
+const waterfall = require('async/waterfall')
+const utils = require('./helpers/utils')
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+
+const expect = chai.expect
+
+describe('test connecting over any relay', function () {
+  this.timeout(500000)
+
+  let portBase = 9010 // TODO: randomize or mock sockets
```

Comment:
@diasdavid I think not randomizing is actually fine, this tests are not ran in parallel so this ports shouldn't conflict with anything, but in rare cases. I think removing the TODO makes sense.